patterns/terraform/main.tf

```diff
@@ -2,15 +2,14 @@ provider "aws" {
   region = local.region
```

Comment:
Don't rename the director to patterns. We want to keep it consistent with other terraform AWS modules. Patterns should go in EKS Blueprints repo.

Reply:
ok, are you saying that we should create multiple different ACK deployment options in the EKS Blueprints repository and keep the "complete" "example" here as an end-to-end test?